apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/GraphQLCompiler.kt

```diff
@@ -46,23 +47,16 @@ class GraphQLCompiler {
       )
     }
 
-    args.transformedQueriesOutputDir?.let { transformedQueriesOutputDir ->
-      if (transformedQueriesOutputDir.exists()) {
-        transformedQueriesOutputDir.deleteRecursively()
+    args.operationOutputFile?.let { operationOutputFile ->
+      val dir = operationOutputFile.parentFile
+      if (dir.exists()) {
+        dir.deleteRecursively()
```

Comment:
Are you sure about deleting the parent directory with all its content? What if the directory is shared with other output? What about just deleting the provided file?

Reply:
Yep, feels safer, will do 👍